.github/ISSUE_TEMPLATE/definition_request.yml

```diff
@@ -0,0 +1,49 @@
+name: Definition Request
+description: Request someone more experienced to create a new flow-typed definition module
+labels: [libdef, request]
+body:
+  - type: input
+    id: dependency
+    attributes:
+      label: Dependency link
+      description: Provide a link to the dependency you'd like typed
+      placeholder: https://www.npmjs.com/package/flow-bin
+    validations:
+      required: true
+
+  - type: input
+    id: version
+    attributes:
+      label: Dependency Version
+      description: What version range do you need typed
+      placeholder: 3.x.x
+    validations:
+      required: true
+
+  - type: input
+    id: flow_version
+    attributes:
+      label: Flow Version
+      description: What minimum flow version do you need this working on
+      placeholder: 0.83.0
+    validations:
+      required: true
```

Comment:
I don't think this should be required, some people won't know what is the significance of this question. I'd rather it not be a barrier to entry, and force people to put in a random value.

Reply:
Yep I'm not fussy there. Done!